<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsMapSaveDialog</class>
 <widget class="QDialog" name="QgsMapSaveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Map as Image</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0">
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Resolution</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QCheckBox" name="mSaveWorldFile">
       <property name="text">
        <string>Append georeference information (embedded or via world file)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QCheckBox" name="mExportMetadataCheckBox">
       <property name="text">
        <string>Export RDF metadata (title, author, etc.)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QgsScaleWidget" name="mScaleWidget" native="true"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QgsExtentGroupBox" name="mExtentGroupBox">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="title">
        <string>Extent</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QgsCollapsibleGroupBoxBasic" name="mGeospatialPDFGroupBox">
       <property name="title">
        <string>Create Geospatial PDF</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QStackedWidget" name="mGeospatialPDFOptionsStackedWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="mGeospatialPdfUnavailableReason">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="1" column="1">
             <widget class="QComboBox" name="mGeospatialPdfFormatComboBox"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Format</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QCheckBox" name="mExportGeospatialPdfFeaturesCheckBox">
              <property name="text">
               <string>Include vector feature information</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QgsCollapsibleGroupBox" name="mAdvancedPdfSettings">
       <property name="title">
        <string>Advanced Settings</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1">
        <item row="2" column="0">
         <widget class="QLabel" name="mTextExportLabel">
          <property name="text">
           <string>Text export</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="mTextRenderFormatComboBox"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="mSimplifyGeometriesCheckbox">
          <property name="text">
           <string>Simplify geometries to reduce output file size</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="mSaveAsRaster">
          <property name="visible">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Advanced effects such as blend modes or vector layer transparency cannot be exported as vectors.
Rasterizing the map is recommended when such effects are used.</string>
          </property>
          <property name="text">
           <string>Rasterize map</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Output height</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QCheckBox" name="mDrawAnnotations">
       <property name="text">
        <string>Draw annotations</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QCheckBox" name="mDrawDecorations">
       <property name="text">
        <string>Draw active decorations</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Output width</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" rowspan="2">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QgsSpinBox" name="mOutputWidthSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="showClearButton" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <layout class="QHBoxLayout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QgsRatioLockButton" name="mLockAspectRatio">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Lock aspect ratio (including while drawing extent onto canvas)</string>
           </property>
           <property name="leftMargin" stdset="0">
            <number>13</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QgsSpinBox" name="mOutputHeightSpinBox">
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="showClearButton" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QgsSpinBox" name="mResolutionSpinBox">
       <property name="suffix">
        <string> dpi</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>3000</number>
       </property>
       <property name="showClearButton" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Scale</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QLabel" name="mInfo">
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsExtentGroupBox</class>
   <extends>QgsCollapsibleGroupBox</extends>
   <header>qgsextentgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsRatioLockButton</class>
   <extends>QToolButton</extends>
   <header>qgsratiolockbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsScaleWidget</class>
   <extends>QWidget</extends>
   <header>qgsscalewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBoxBasic</class>
   <extends>QGroupBox</extends>
   <header location="global">qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mExtentGroupBox</tabstop>
  <tabstop>mResolutionSpinBox</tabstop>
  <tabstop>mOutputWidthSpinBox</tabstop>
  <tabstop>mLockAspectRatio</tabstop>
  <tabstop>mOutputHeightSpinBox</tabstop>
  <tabstop>mDrawDecorations</tabstop>
  <tabstop>mDrawAnnotations</tabstop>
  <tabstop>mSaveWorldFile</tabstop>
  <tabstop>mExportMetadataCheckBox</tabstop>
  <tabstop>mGeospatialPDFGroupBox</tabstop>
  <tabstop>mGeospatialPdfFormatComboBox</tabstop>
  <tabstop>mExportGeospatialPdfFeaturesCheckBox</tabstop>
  <tabstop>mSaveAsRaster</tabstop>
  <tabstop>mSimplifyGeometriesCheckbox</tabstop>
  <tabstop>mTextRenderFormatComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsMapSaveDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsMapSaveDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
